### ISSUE TITLE
Add friendly variables in template

### ISSUE THREAD

**Issue by sylvainvivien** [Arachnid/bloggart#21]
For people that "only" want to create a theme, I think these django tags are not very friendly and they have to check the code to understand them.

{% ifequal generator_class "IndexContentGenerator" %}
{% ifequal handler_class "AdminHandler" %}
{% ifequal path "/admin/newpost" %}

Maybe this could be replace with nice and easy variable :

{% if index %}
{% if admin %}
{% if new_post %}
etc,...

then it could be nice to create all :

{% if atom %} #atom generator
{% if edit_post %} #edit a post
{% if tag %} #tag generator
etc,....

**Comment by Arachnid** [comment]
I agree with the sentiment, but I'm not so sure this is a practical way to handle it. It needs thought.
